docs/get-started/json-rpc-commands.mdx

```diff
@@ -0,0 +1,879 @@
+
+### eth_getFilterChanges
+
+Polling method for a filter, which returns an array of logs which occurred since last poll.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+*  <b>  QUANTITY </b> - the filter id.
+
+<h4><i>Returns:</i></h4>
+<b> Array </b> - Array of log objects, or an empty array if nothing has changed since last poll.
+
+*  For filters created with eth_newBlockFilter the return are block hashes (DATA, 32 Bytes), e.g. ["0x3454645634534..."].
+*  For filters created with eth_newPendingTransactionFilter  the return are transaction hashes (DATA, 32 Bytes), e.g. ["0x6345343454645..."].
+*  For filters created with eth_newFilter logs are objects with following params:
+    * <b> removed: TAG </b> - true when the log was removed, due to a chain reorganization. false if its a valid log.
+    * <b> logIndex: QUANTITY </b> - integer of the log index position in the block. null when its pending log.
+    * <b> transactionIndex: QUANTITY </b> - integer of the transactions index position log was created from. null when its pending log.
+    * <b> transactionHash: DATA, 32 Bytes </b> - hash of the transactions this log was created from. null when its pending log.
+    * <b> blockHash: DATA, 32 Bytes </b> - hash of the block where this log was in.  null when its pending log.
+    * <b> blockNumber: QUANTITY </b> - the block number where this log was in.  null when its pending log.
+    * <b> address: DATA, 20 Bytes </b> - address from which this log originated.
+    * <b> data: DATA </b> - contains one or more 32 Bytes non-indexed arguments of the log.
+    * <b> topics: Array of DATA </b> - Array of 0 to 4 32 Bytes DATA of indexed log arguments. (In solidity: The first topic is the hash of the signature of the event (e.g. Deposit(address,bytes32,uint256)), except you declared the event with the anonymous specifier.)
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getFilterChanges","params":["0x16"],"id":1}'
+````
+
+### eth_uninstallFilter
+
+Uninstalls a filter with given id. Should always be called when watch is no longer needed.
+Additonally Filters timeout when they aren’t requested with eth_getFilterChanges for a period of time.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+*  <b> QUANTITY </b> - The filter id.
+
+<h4><i>Returns:</i></h4>
+
+*  <b> Boolean </b> - true if the filter was successfully uninstalled, otherwise false.
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_uninstallFilter","params":["0xb"],"id":1}'
+````
+
+### eth_unsubscribe
+
+Subscriptions are cancelled with a regular RPC call with eth_unsubscribe as method and the subscription id as first parameter. It returns a bool indicating if the subscription was cancelled successful.
```

Comment:
change to "Subscriptions are canceled with a regular RPC call with eth_unsubscribe as a method and the subscription id as the first parameter. It returns a bool indicating if the subscription was canceled successfully."

Reply:
Fixed! 
 [b1173b4](https://github.com/0xPolygon/polygon-sdk-docs/pull/41/commits/b1173b4db2a92b5f3c153e1906ad8ac9e82900bf)